### ISSUE TITLE
Make navbar responsive for smaller screens.

### ISSUE THREAD

**Issue by karstenpedersen** [karstenpedersen/obsidian-daily-note-navbar#16]
Hide some of the days if there isn't enough room.

We could recommend to use Calendar plugin if there isn't enough room on the screen to even show our navbar.

**Comment by xRyul** [comment]
Would be nice! Although on iPhone I already can see only 2 or 3 days. And can't see the arrows left or right, thus unable to navigate any days. And if I try to swipe the navbar left or right, it just triggers default obsidian side panels to appear. Thus it would be nice to either drop whole Navbar 1 line down, so that at least user can see the arrows for navigation or add an option for the calendar picker to appear.

**Comment by karstenpedersen** [comment]
> Would be nice! Although on iPhone I already can see only 2 or 3 days. And can't see the arrows left or right, thus unable to navigate any days. And if I try to swipe the navbar left or right, it just triggers default obsidian side panels to appear. Thus it would be nice to either drop whole Navbar 1 line down, so that at least user can see the arrows for navigation or add an option for the calendar picker to appear.

Yeah this is also what I am thinking. I have not made it so you can drag it, so it makes sense you can't. Good idea with dropping it down.

**Comment by chadmiller-saq** [comment]
I propose that under some width, it abandon the seven-day week view and instead change to a five-day and then three-day current-centered contextual view.

**Comment by focusaurus** [comment]
My suggestion for narrow screens like a phone in portrait mode would be three buttons: previous, calendar, next. The calendar button would open the current month in a flyover.

**Comment by focusaurus** [comment]
I forked this plugin and was able to get 3 buttons for yesterday, today, and a month calendar popup. What do folks think about this?

<img width="760" height="401" alt="Image" src="https://github.com/user-attachments/assets/6dd675ec-1fce-432c-8a67-12cac88492a7" />
